src/Sylius/Behat/Client/ResponseChecker.php

```diff
@@ -68,6 +68,12 @@ public function hasValue(Response $response, string $key, $value): bool
         return $this->getResponseContentValue($response, $key) === $value;
     }
 
+    /** @param string|int $value */
+    public function hasValueInCollection(Response $response, string $key, $value): bool
```

Comment:
We should definitely refactor names of these methods in future, as I thought at the first moment that this is the same method as `hasItemWithValue`